src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/yml/messenger.yml

```
@@ -1,5 +1,6 @@
 framework:
     messenger:
+        stop_signals: SIGTERM
```

Why does this have to be plural (a list of signals) instead of just 1 signal?

```suggestion
        stop_signal: SIGTERM
```

---

in case project is used in several envs (docker FPM alpine environment, and classical VMs with supervisor) where signals could be different.

---

Then configure the correct signal for that env. Doesn't make sense (to me) to just fire all the signals.

---

Agreed with @Warxcell. we should allow multiple signals.